isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -872,55 +1653,186 @@ namespace Isis {
+            switch(i) {
+
+            case 1:
+              parameterNamesList.append( str.arg("     ").arg("dd/s") );
+              correctionUnitList.append("dd");
+              break;
+            case 2:
+              parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+              correctionUnitList.append("dd/s^2");
+              break;
+            case 3:
+              parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+              correctionUnitList.append("dd/s^3");
+              break;
+            default:
+                parameterNamesList.append(str.arg("     ").arg("dd/s^"+toString(i) ) );
+                correctionUnitList.append("dd/s^"+toString(i));
+
+            }
+          }
         }
         for (int i = 0; i < nPointingCoefficients; i++) {
           finalParameterValues.push_back(coefDEC[i] * RAD2DEG);
-          if (i == 0)
-            parameterNamesList.append( str.arg("DEC  ").arg("0") );
-          else
-            parameterNamesList.append( str.arg("     ").arg(i) );
+          if (i == 0) {
+            parameterNamesList.append( str.arg("DEC  ").arg("dd") );
+            correctionUnitList.append("dd");
+
+          }
+          else {
+
+            switch(i) {
+
+            case 1:
+              parameterNamesList.append( str.arg("     ").arg("dd/s") );
+              correctionUnitList.append("dd/s");
+              break;
+            case 2:
+              parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+              correctionUnitList.append("dd/s^2");
+              break;
+            case 3:
+              parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+              correctionUnitList.append("dd/s^3");
+              break;
+            default:
+                parameterNamesList.append(str.arg("     ").arg("dd/s^"+toString(i) ) );
+                correctionUnitList.append("dd/s^"+toString(i) );
+
+            }
+          }
         }
         for (int i = 0; i < nPointingCoefficients; i++) {
           finalParameterValues.push_back(coefTWI[i] * RAD2DEG);
-          if (i == 0)
-            parameterNamesList.append( str.arg("TWI  ").arg("0") );
-          else
-            parameterNamesList.append( str.arg("     ").arg(i) );
+          if (i == 0) {
+            parameterNamesList.append( str.arg("TWI  ").arg("dd") );
+            correctionUnitList.append("dd");
+            }            
+          else {
+            switch(i) {
+            case 1:
+              parameterNamesList.append( str.arg("     ").arg("dd/s") );
+              correctionUnitList.append("dd/s");
+              break;
+            case 2:
+              parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+              correctionUnitList.append("dd/s^2");
+              break;
+            case 3:
+              parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+              correctionUnitList.append("dd/s^3");
+              break;
```

Comment:
Remove case 2 and 3

Reply:
probably worth just saying et cetera so the PR doesn't get saturated with comments